android/src/main/java/com/beefe/picker/PickerViewModule.java

```diff
@@ -188,7 +188,7 @@ public void _init(ReadableMap options) {
                 int[] colors = getColor(array);
                 confirmTV.setTextColor(argb(colors[3], colors[0], colors[1], colors[2]));
             }
-            confirmTV.setOnClickListener(new View.OnClickListener() {
+            barLayout.setOnClickListener(new View.OnClickListener() {
```

Comment:
Since `confirmTV` is on top of `barLayout`, does clicking `confirmTV` clobber the click event now?